api/src/main/java/com/valtech/aem/saas/api/bestbets/BestBetsConsumerService.java

```diff
@@ -0,0 +1,67 @@
+package com.valtech.aem.saas.api.bestbets;
+
+import java.util.List;
+import lombok.NonNull;
+
+/**
+ * Represents the best bets consumer service. Offers the following actions: - add a single best bet - add multiple best
+ * bets - update the best bet - delete a best bet - publish best bets for project - get all best bets
+ */
+public interface BestBetsConsumerService {
```

Comment:
- [ ] Can you rename to BestBetsService? I don't see why we need the word Consumer, all service will be consumed